package.json

```diff
@@ -44,26 +44,26 @@
   },
   "devDependencies": {
     "@ssv/tools": "^0.8.1",
-    "@types/jest": "^24.0.19",
-    "del": "^4.1.1",
+    "@types/jest": "^26.0.4",
+    "del": "^5.1.0",
     "gulp": "^4.0.2",
-    "gulp-bump": "^3.1.3",
-    "gulp-conventional-changelog": "^2.0.19",
+    "gulp-bump": "^3.2.0",
+    "gulp-conventional-changelog": "^2.0.32",
     "gulp-util": "^3.0.8",
-    "jest": "^24.8.0",
-    "jest-cli": "^24.8.0",
+    "jest": "^26.1.0",
+    "jest-cli": "^26.1.0",
     "jest-environment-node-debug": "^2.0.0",
-    "microbundle": "^0.11.0",
+    "microbundle": "^0.12.3",
     "require-dir": "^1.2.0",
-    "rxjs": "^6.5.3",
-    "ts-jest": "^24.1.0",
-    "typescript": "^3.6.4",
-    "typescript-tslint-plugin": "^0.5.4",
-    "yargs": "^14.2.0"
+    "rxjs": "^6.6.0",
+    "ts-jest": "^26.1.2",
+    "typescript": "^3.9.6",
+    "typescript-tslint-plugin": "^0.5.5",
+    "yargs": "^15.4.1"
   },
   "dependencies": {
-    "@microsoft/signalr": "~3.0.0",
-    "tslib": "^1.10.0"
+    "@microsoft/signalr": "~3.1.6",
+    "tslib": "^2.0.0"
```

Comment:
tslib 2 the consumer needs to use ts3.9+ https://github.com/microsoft/tslib/releases

so ideally its done when its a breaking change